src/chi-vue/src/components/data-table-filters/AdvancedFilters.tsx

```diff
@@ -287,6 +288,13 @@ export default class AdvancedFilters extends Vue {
             modal={this._chiMajorVersion === 5}
             drag
             closable>
+            <div class={`${UTILITY_CLASSES.MARGIN.BOTTOM[1]}`}>
+              <button
+                class={`${BUTTON_CLASSES.BUTTON} ${BUTTON_CLASSES.FLAT} ${BUTTON_CLASSES.PRIMARY} ${BUTTON_CLASSES.SIZES.SM} ${BUTTON_CLASSES.NO_HOVER} ${UTILITY_CLASSES.PADDING.X[0]} ${UTILITY_CLASSES.TYPOGRAPHY.TEXT_NORMAL}`}
+                onclick={(event: any) => this._expandCollapseAccordions(event)}>
```

Comment:
```suggestion
                onclick={(event: Event) => this._expandCollapseAccordions(event)}>
```
Shouldn't it be `Event` instead of `any`?

Reply:
Yes, we can change it to Event to be more specific.